api/Onboard.go

```diff
@@ -0,0 +1,46 @@
+package api
+
+import (
+	"fmt"
+	"github.com/operate-first/opfcli/models"
+	"strings"
+)
+
+func (api *API) Onboard(path string, projectDisplayName string, disableLimitrange bool) error {
+	onboardRequest, err := models.OnboardRequestFromYAMLPath(path)
+
+	if err := api.CreateGroup(
+		onboardRequest.Teamname,
+		true,
+	); err != nil {
+		return err
+	}
+
+	if err := api.CreateRoleBinding(onboardRequest.Teamname, "admin"); err != nil {
+		return err
+	}
+
+	var tempQuota string
+	for i := 0; i < len(onboardRequest.Namespaces); i++ {
```

Comment:
use `range` instead: 
`for _, ns := range onboardingTemplate.Namespaces` 
it's much neater